src/anshuKJun20/DuplicateNumber.java

```diff
@@ -0,0 +1,20 @@
+package anshuKJun20;
+
+public class DuplicateNumber {
+	int value;
```

Comment:
Actually what I was expecting is don't initialize any variable.As we are not returning or reusing duplicate number try to optimize it .
for (int i= 0; i< input.length; i++) {
			for (int j= i+1; j< input.length; j++) {
				if (input[i] == input[j]) {
					System.out.println("\nDuplicate Number is: " + input[i]);
					break;

* If you want to initialize variable then needs to be local variable.no need of instance variable.

Reply:
Changes done .